.codacy.yml

```diff
@@ -0,0 +1,3 @@
+# ignore files auto generated by Django
+exclude_paths:
+  - 'wellregistry/registry/migrations/**'
```

Comment:
I would prefer not adding this. Some migrations are not auto generated (for instance 0002_add_agency_groups.py. Please remove this file.

Reply:
While that is true that they are not all auto generated, that all start with numbers which means they will always report an issue. I would leave it in.

Reply:
That would exclude the file for any kind of linting so I would like this removed.